tasks/service-runner/config/concise-sidecar.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: concise
+  labels:
+    name: concise
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: concise
+  template:
+    metadata:
+      labels:
+        name: concise
+    spec:
+      containers:
+        - name: worker
+          imagePullPolicy: $PODAAC_CONCISE_IMAGE_PULL_POLICY
+          image: $PODAAC_CONCISE_IMAGE
+          resources:
+            limits:
+              memory: $PODAAC_CONCISE_LIMITS_MEMORY
+            requests:
+              memory: $PODAAC_CONCISE_REQUESTS_MEMORY
+          envFrom:
+          - configMapRef:
+              name: harmony-env
+          - secretRef:
+              name: harmony-secrets
+          env:
+            - name: TEXT_LOGGER
+              value: "false"
+          volumeMounts:
+            - mountPath: /tmp/metadata
+              name: test-volume
+          command: ["/bin/sh", "-c"]
+          args:
+            - while true; do
+                date;
+                sleep 10;
+              done
+        - name: manager
+          imagePullPolicy: IfNotPresent
+          image: $SERVICE_RUNNER_IMAGE
+          resources:
+            limits:
+              memory: $SERVICE_RUNNER_LIMITS_MEMORY
+            requests:
+              memory: $SERVICE_RUNNER_REQUESTS_MEMORY
+          envFrom:
+          - configMapRef:
+              name: harmony-env
+          - secretRef:
+              name: harmony-secrets
+          env:
+          - name: BACKEND_HOST
+            value: $BACKEND_HOST
+          - name: BACKEND_PORT
+            value: "$BACKEND_PORT"
+          - name: HARMONY_SERVICE
+            value: $PODAAC_CONCISE_IMAGE
+          - name: WORKING_DIR
+            value: "/opt/harmony-concise"
+          - name: INVOCATION_ARGS
+            value: |-
+              concise_harmony
```

Comment:
I understand why this is necessary; however, it seems like we're moving toward needing more one-off config per service intermingled with boilerplate.  For the "Definitely some day, but not in this ticket" I'd like us to be trying to put the onus for setting service-specific values on service providers (and making that as low-effort as possible) while getting the boilerplate centralized / generated.